fetch.bs

```diff
@@ -4948,19 +4949,22 @@ constructor must run these steps:
    to <var>method</var>.
   </ol>
 
- <li><p>Let <var>r</var> be a new {{Request}} object associated with <var>request</var> and
- a new associated {{Headers}} object whose <a for=Headers>guard</a>
- is "<code>request</code>".
+ <li><p>Let <var>r</var> be a new {{Request}} object associated with <var>request</var>.
+
+ <li><p>Let <var>headersObject</var> be a new {{Headers}} object, whose
+ <a for=Headers>header list</a> is <var>request</var>'s <a for=request>header list</a>, and
+ <a for=Headers>guard</a> is "<code>request</code>".
 
- <li><p>Let <var>headers</var> be a copy of <var>r</var>'s {{Headers}} object and its
+ <li><p>Set <var>r</var>'s <a for=Request>headers</a> to <var>headersObject</var>.
```

Comment:
Maybe merge this with the previous step? Set r's headers to a new Headers object, ... And then we don't need to use _headersObject_.